tests/Jwt/FirebaseParserTest.php

```diff
@@ -51,32 +50,28 @@ protected function setUp()
         ];
     }
 
+    /**
+     * @expectedException \Equip\Auth\Exception\InvalidException
+     * @expectedExceptionCode \Equip\Auth\Exception\InvalidException::CODE_TOKEN_EXPIRED
```

Comment:
@AlexMasterov Fair enough, though dropping 5.5 support may be something we want to consider doing given that it's going EOL next month.